integreat_cms/cms/fixtures/test_data.json

```diff
@@ -3062,1742 +3841,2597 @@
+      "language": 1,
+      "currently_in_translation": false,
+      "machine_translated": false,
+      "version": 3,
+      "minor_edit": false,
+      "last_updated": "2022-02-22T16:57:08.587Z",
+      "creator": ["root"],
+      "automatic_translation": false,
+      "page": 1
     }
   },
   {
-    "model": "cms.pushnotification",
+    "model": "cms.imprintpagetranslation",
     "pk": 5,
     "fields": {
-      "regions": [1],
-      "channel": "news",
-      "draft": true,
-      "is_template": true,
-      "template_name": "Vorlage",
-      "sent_date": null,
-      "created_date": "2023-10-09T20:15:00.000Z",
-      "mode": "USE_MAIN_LANGUAGE"
-    }
-  },
-  {
-    "model": "cms.pushnotification",
-    "pk": 6,
-    "fields": {
-      "regions": [2],
-      "channel": "news",
-      "draft": false,
-      "sent_date": "2022-03-05T10:32:33.118Z",
-      "created_date": "2022-03-05T10:31:33.118Z",
-      "mode": "USE_MAIN_LANGUAGE"
-    }
-  },
-  {
-    "model": "cms.pushnotification",
-    "pk": 7,
-    "fields": {
-      "regions": [4],
-      "channel": "news",
-      "draft": false,
-      "sent_date": "2022-03-05T10:32:33.118Z",
-      "created_date": "2022-03-05T10:31:33.118Z",
-      "mode": "USE_MAIN_LANGUAGE"
+      "title": "Rechtliches",
+      "status": "AUTO_SAVE",
+      "content": "<p>Autosave</p>",
+      "language": 1,
+      "currently_in_translation": false,
+      "machine_translated": false,
+      "version": 4,
+      "minor_edit": false,
+      "last_updated": "2023-10-11T16:57:08.587Z",
+      "creator": ["root"],
+      "automatic_translation": false,
+      "page": 1
     }
   },
+  { "model": "cms.eventfeedback", "pk": 3, "fields": { "feedback_ptr": 3, "event_translation": 1 } },
```

Comment:
```suggestion
```

Reply:
I would like to avoid manually changing test data, because that would make it very hard to add new test data later (We would have to manually type out the new test data, or automatically generate it and the apply your suggested changes every time). 
I have searched a bit and found this issue: https://code.djangoproject.com/ticket/8399
There, the proposed solution seems to add a check to signal handlers for a `raw` argument and return if that is True. We could add a decorator that does this to our signal handlers. Would you be okay with that?

Reply:
For example, if feedback_create_handler() is executed after "cms.eventfeedback" is created, but before "cms.feedback", and we just "pass", perhaps this would cause inconsistent feedback state in the cache, or something ?

Reply:
> We could add a decorator that does this to our signal handlers. Would you be okay with that?

The concept sounds good to me

Reply:
I think that should be fine. This specific code is only relevant when we update an existing feedback. But in general, if we ever find issues with that, we could just invalidate the entire cache after loading test data (Maybe that even happens already?)

Reply:
I also just found this note in the django documentation which explicitly states that we should not query other models during loaddata: https://docs.djangoproject.com/en/5.0/ref/signals/#pre-save